### DISCUSSION TITLE
Make fluid  wallpaper transparent, transparent.

### DISCUSSION THREAD

**Discussion by 42willow** [rocksdanister/lively#348] [Ideas]
The "BG Transparent" option in Fluids V2 just changes the background to a bunch of squares
![image](https://user-images.githubusercontent.com/70784752/110901140-f32ae480-8357-11eb-9bf2-efb39a1437f2.png)

It would be good if the squares were replaced with your pre-existing wallpaper, if you could add a "BG Transparent" option to the others like Fluids V2 then you could change the mountain with a burger for example, that would be really cool.

**Comment by rocksdanister** [comment]
You will have to edit the [webpage code](https://github.com/rocksdanister/WebGL-Fluid-Simulation) and change the texture to custom image file. 
Optionally use [Folder dropdown](https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction#folder-dropdown) control to change the file during runtime.

**Reply by 42willow** [reply]
Thanks, I am a bit confused about the "Folder dropdown" control but I will try changing the webpage code.

**Reply by rocksdanister** [reply]
>I am a bit confused about the "Folder dropdown" control

Its used in the `Rain v2` wallpaper example:

https://github.com/rocksdanister/shadertoy-heartfelt/blob/1bf23aec7855ddf418a46c035710e1d3f166c26b/LivelyProperties.json#L47

https://github.com/rocksdanister/shadertoy-heartfelt/blob/1bf23aec7855ddf418a46c035710e1d3f166c26b/index.html#L64

**Reply by 42willow** [reply]
¯\_(ツ)_/¯ Will give it a go, thanks. Great app by the way.

**Comment by rocksdanister** [comment]
For anyone interested, this version supports background image:
https://github.com/rocksdanister/WebGL-Fluid-Simulation/releases/tag/v3

**Comment by erwinwallpaper** [comment]
Nice, new features.
